src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -517,7 +517,7 @@ protected override void ProcessRecord()
 
                 bool handleRedirect = keepAuthorizationOnRedirect || AllowInsecureRedirect || PreserveHttpMethodOnRedirect;
 
-                using (HttpClient client = GetHttpClient(handleRedirect))
+                var client = GetHttpClient(handleRedirect);
```

Comment:
Please replace `var` with real type for readability.

Reply:
Done

Reply:
You could also remove { } and one level of indentation

Reply:
I deliberately backed out that change as I thought it would make later merges more difficult and also inflate the size of the PR and make it harder to review. I can still do it if you think it is best.

Reply:
Don't worry I'll do it in a follow up PR